{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "id": "y0PDI84iSiHe"
   },
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "from urllib.request import urlopen\n",
    "from bs4 import BeautifulSoup as soup\n",
    "import requests\n",
    "from bs4 import BeautifulSoup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 72
    },
    "id": "Hl2E7rNMSpwu",
    "outputId": "2779fcdc-21cd-440e-86d0-da12b5b6d6de"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13827', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13830', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13832', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57709', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13992', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14147', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57681', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57684', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57718', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13996', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57740', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=193115', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13672', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13818', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13820', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14246', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170586', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57745', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13985', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=16017', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=37589', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=37592', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14148', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57690', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57749', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170593', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170594', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170597', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170598', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14190', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170601', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170602', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170604', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170605', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14186', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57696', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57730', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57752', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170609', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57756', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14149', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14150', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14192', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14193', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57732', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170613', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170616', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14006', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14151', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=193118', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170221', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=57763', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13906', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14008', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=14152', 'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=170222']\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=13827'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a=\"\"\"<area shape=\"rect\" coords=\"59,140.9,67,128.9\" href=\"qdetails?QTL_ID=13827\">\n",
    " <area shape=\"rect\" coords=\"72,174.7,80,162.7\" href=\"qdetails?QTL_ID=13830\">\n",
    " <area shape=\"rect\" coords=\"85,163.5,93,151.5\" href=\"qdetails?QTL_ID=13832\">\n",
    " <area shape=\"rect\" coords=\"98,294.51,106,282.51\" href=\"qdetails?QTL_ID=57709\">\n",
    " <area shape=\"rect\" coords=\"119,37,127,393.2\" href=\"chromap?chromos=2&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"109,36,147,23\" href=\"chromap?chromos=2&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"132,286.7,140,274.7\" href=\"qdetails?QTL_ID=13992\">\n",
    " <area shape=\"rect\" coords=\"145,275.4,153,263.4\" href=\"qdetails?QTL_ID=14147\">\n",
    " <area shape=\"rect\" coords=\"158,108.74,166,96.74\" href=\"qdetails?QTL_ID=57681\">\n",
    " <area shape=\"rect\" coords=\"171,257.39,179,245.39\" href=\"qdetails?QTL_ID=57684\">\n",
    " <area shape=\"rect\" coords=\"184,109.52,192,97.52\" href=\"qdetails?QTL_ID=57718\">\n",
    " <area shape=\"rect\" coords=\"192,37,200,380.3\" href=\"chromap?chromos=3&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"182,36,220,23\" href=\"chromap?chromos=3&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"205,152,213,140\" href=\"qdetails?QTL_ID=13996\">\n",
    " <area shape=\"rect\" coords=\"218,169.99,226,157.99\" href=\"qdetails?QTL_ID=57740\">\n",
    " <area shape=\"rect\" coords=\"265,37,273,225.8\" href=\"chromap?chromos=4&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"255,36,293,23\" href=\"chromap?chromos=4&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"278,153.67,286,141.67\" href=\"qdetails?QTL_ID=193115\">\n",
    " <area shape=\"rect\" coords=\"338,37,346,236.9\" href=\"chromap?chromos=5&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"328,36,366,23\" href=\"chromap?chromos=5&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"411,37,419,233.7\" href=\"chromap?chromos=6&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"401,36,439,23\" href=\"chromap?chromos=6&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"424,169.16,432,157.16\" href=\"qdetails?QTL_ID=13672\">\n",
    " <area shape=\"rect\" coords=\"437,96.61,445,84.61\" href=\"qdetails?QTL_ID=13818\">\n",
    " <area shape=\"rect\" coords=\"450,96.61,458,84.61\" href=\"qdetails?QTL_ID=13820\">\n",
    " <area shape=\"rect\" coords=\"463,87.53,471,75.53\" href=\"qdetails?QTL_ID=14246\">\n",
    " <area shape=\"rect\" coords=\"476,39,484,27\" href=\"qdetails?QTL_ID=170586\">\n",
    " <area shape=\"rect\" coords=\"484,37,492,226.4\" href=\"chromap?chromos=7&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"474,36,512,23\" href=\"chromap?chromos=7&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"497,99.5,505,87.5\" href=\"qdetails?QTL_ID=57745\">\n",
    " <area shape=\"rect\" coords=\"557,37,565,205.8\" href=\"chromap?chromos=8&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"547,36,585,23\" href=\"chromap?chromos=8&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"630,37,638,204.9\" href=\"chromap?chromos=9&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"620,36,658,23\" href=\"chromap?chromos=9&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"643,55.6,651,43.6\" href=\"qdetails?QTL_ID=13985\">\n",
    " <area shape=\"rect\" coords=\"703,37,711,178.2\" href=\"chromap?chromos=10&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"693,36,731,23\" href=\"chromap?chromos=10&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"265,297,273,447.6\" href=\"chromap?chromos=11&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"255,296,293,283\" href=\"chromap?chromos=11&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"278,365.6,286,353.6\" href=\"qdetails?QTL_ID=16017\">\n",
    " <area shape=\"rect\" coords=\"291,382.71,299,370.71\" href=\"qdetails?QTL_ID=37589\">\n",
    " <area shape=\"rect\" coords=\"304,382.7,312,370.7\" href=\"qdetails?QTL_ID=37592\">\n",
    " <area shape=\"rect\" coords=\"338,297,346,444.4\" href=\"chromap?chromos=12&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"328,296,366,283\" href=\"chromap?chromos=12&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"351,353.2,359,341.2\" href=\"qdetails?QTL_ID=14148\">\n",
    " <area shape=\"rect\" coords=\"364,390.57,372,378.57\" href=\"qdetails?QTL_ID=57690\">\n",
    " <area shape=\"rect\" coords=\"411,297,419,466.3\" href=\"chromap?chromos=13&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"401,296,439,283\" href=\"chromap?chromos=13&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"424,319.28,432,307.28\" href=\"qdetails?QTL_ID=57749\">\n",
    " <area shape=\"rect\" coords=\"437,299,445,287\" href=\"qdetails?QTL_ID=170593\">\n",
    " <area shape=\"rect\" coords=\"450,299,458,287\" href=\"qdetails?QTL_ID=170594\">\n",
    " <area shape=\"rect\" coords=\"463,299,471,287\" href=\"qdetails?QTL_ID=170597\">\n",
    " <area shape=\"rect\" coords=\"476,299,484,287\" href=\"qdetails?QTL_ID=170598\">\n",
    " <area shape=\"rect\" coords=\"484,297,492,456.9\" href=\"chromap?chromos=14&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"474,296,512,283\" href=\"chromap?chromos=14&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"497,363.29,505,351.29\" href=\"qdetails?QTL_ID=14190\">\n",
    " <area shape=\"rect\" coords=\"510,299,518,287\" href=\"qdetails?QTL_ID=170601\">\n",
    " <area shape=\"rect\" coords=\"523,299,531,287\" href=\"qdetails?QTL_ID=170602\">\n",
    " <area shape=\"rect\" coords=\"536,299,544,287\" href=\"qdetails?QTL_ID=170604\">\n",
    " <area shape=\"rect\" coords=\"549,299,557,287\" href=\"qdetails?QTL_ID=170605\">\n",
    " <area shape=\"rect\" coords=\"557,297,565,461.8\" href=\"chromap?chromos=15&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"547,296,585,283\" href=\"chromap?chromos=15&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"570,381.05,578,369.05\" href=\"qdetails?QTL_ID=14186\">\n",
    " <area shape=\"rect\" coords=\"630,297,638,422.7\" href=\"chromap?chromos=16&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"620,296,658,283\" href=\"chromap?chromos=16&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"643,337.08,651,325.08\" href=\"qdetails?QTL_ID=57696\">\n",
    " <area shape=\"rect\" coords=\"656,314.19,664,302.19\" href=\"qdetails?QTL_ID=57730\">\n",
    " <area shape=\"rect\" coords=\"669,315.4,677,303.4\" href=\"qdetails?QTL_ID=57752\">\n",
    " <area shape=\"rect\" coords=\"682,299,690,287\" href=\"qdetails?QTL_ID=170609\">\n",
    " <area shape=\"rect\" coords=\"703,297,711,468\" href=\"chromap?chromos=17&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"693,296,731,283\" href=\"chromap?chromos=17&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"716,375.23,724,363.23\" href=\"qdetails?QTL_ID=57756\">\n",
    " <area shape=\"rect\" coords=\"46,527,54,695.7\" href=\"chromap?chromos=18&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"36,526,74,513\" href=\"chromap?chromos=18&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"59,585,67,573\" href=\"qdetails?QTL_ID=14149\">\n",
    " <area shape=\"rect\" coords=\"72,585,80,573\" href=\"qdetails?QTL_ID=14150\">\n",
    " <area shape=\"rect\" coords=\"85,636.79,93,624.79\" href=\"qdetails?QTL_ID=14192\">\n",
    " <area shape=\"rect\" coords=\"98,636.79,106,624.79\" href=\"qdetails?QTL_ID=14193\">\n",
    " <area shape=\"rect\" coords=\"119,527,127,639.8\" href=\"chromap?chromos=19&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"109,526,147,513\" href=\"chromap?chromos=19&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"132,587.5,140,575.5\" href=\"qdetails?QTL_ID=57732\">\n",
    " <area shape=\"rect\" coords=\"145,529,153,517\" href=\"qdetails?QTL_ID=170613\">\n",
    " <area shape=\"rect\" coords=\"158,529,166,517\" href=\"qdetails?QTL_ID=170616\">\n",
    " <area shape=\"rect\" coords=\"192,527,200,667.9\" href=\"chromap?chromos=20&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"182,526,220,513\" href=\"chromap?chromos=20&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"205,582.9,213,570.9\" href=\"qdetails?QTL_ID=14006\">\n",
    " <area shape=\"rect\" coords=\"218,598.5,226,586.5\" href=\"qdetails?QTL_ID=14151\">\n",
    " <area shape=\"rect\" coords=\"231,595.88,239,583.88\" href=\"qdetails?QTL_ID=193118\">\n",
    " <area shape=\"rect\" coords=\"265,527,273,643.5\" href=\"chromap?chromos=21&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"255,526,293,513\" href=\"chromap?chromos=21&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"278,572.59,286,560.59\" href=\"qdetails?QTL_ID=170221\">\n",
    " <area shape=\"rect\" coords=\"338,527,346,650.9\" href=\"chromap?chromos=22&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"328,526,366,513\" href=\"chromap?chromos=22&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"351,539.72,359,527.72\" href=\"qdetails?QTL_ID=57763\">\n",
    " <area shape=\"rect\" coords=\"411,527,419,651.9\" href=\"chromap?chromos=23&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"401,526,439,513\" href=\"chromap?chromos=23&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"424,612.9,432,600.9\" href=\"qdetails?QTL_ID=13906\">\n",
    " <area shape=\"rect\" coords=\"484,527,492,649.6\" href=\"chromap?chromos=24&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"474,526,512,513\" href=\"chromap?chromos=24&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"497,602,505,590\" href=\"qdetails?QTL_ID=14008\">\n",
    " <area shape=\"rect\" coords=\"510,552,518,540\" href=\"qdetails?QTL_ID=14152\">\n",
    " <area shape=\"rect\" coords=\"557,527,565,636.3\" href=\"chromap?chromos=25&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"547,526,585,513\" href=\"chromap?chromos=25&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"630,527,638,639.1\" href=\"chromap?chromos=26&amp;trait_ID=3039&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"620,526,658,513\" href=\"chromap?chromos=26&amp;trait_ID=3039\">\n",
    " <area shape=\"rect\" coords=\"643,545.02,651,533.02\" href=\"qdetails?QTL_ID=170222\">\"\"\"\n",
    "\n",
    "\n",
    "L=[]\n",
    "#for num in a:\n",
    "\n",
    "f=\"qdetails?QTL_ID=\"\n",
    "fil=len(f)\n",
    "\n",
    "for i in range(0,len(a)-fil):\n",
    "    \n",
    "    if a[i:i+fil]==f:  #el ultimo no lo toma\n",
    "        L.append(a[i+fil:i+fil+7])\n",
    "\n",
    "Link=[]\n",
    "for i in L:\n",
    "    if i[len(i)-1]==\">\":\n",
    "        Link.append(i[0:5])\n",
    "\n",
    "    else:\n",
    "        Link.append(i[0:6])\n",
    "        \n",
    "    \n",
    "#print(Link)\n",
    "\n",
    "LINK=[]\n",
    "url=\"https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=\"\n",
    "for i in Link:\n",
    "    LINK.append(url+i)\n",
    "print(LINK)\n",
    "LINK[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 89
    },
    "id": "qBEZwSPsSxKs",
    "outputId": "ddd541b1-b2ce-4a4e-e1b5-337df78b7aff"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[[], ['Chromosome:', '1'], ['QTL Peak Location:', '96.42 (cM)'], ['QTL Span:', '95.9-101.9 (cM)78.3-85.2 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'INRA3'], ['Peak:', ''], ['Lower, \"Significant\":', 'BM4129'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', '-'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.0172'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Manchega sheep.'], ['Breeds associated:\\n  \\nManchega'], ['Design:', 'A daughter design was used.  Animals were genotyped for polymorphisms in alpha-amylase and analyzed for milk traits.'], ['Analysis:', 'The general linear model procedure was used.'], ['Software:', 'CRIMAP, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Calvo JH, Marcos S, Beattie AE, Gonzalez C, Jurado JJ, Serrano M'], ['Affiliation:', 'Departamento de Mejora Gen�tica Animal, INIA, 28040 Madrid, Spain'], ['Title:', 'Ovine alpha-amylase genes: isolation, linkage mapping and association analysis with milk traits'], ['Journal:', 'Animal genetics, 2004, 35(4): 329-32'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '1'], ['QTL Peak Location:', '128.2 (cM)'], ['QTL Span:', '124.5-135.7 (cM)97.9-109.6 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BMS482'], ['Peak:', ''], ['Lower, \"Significant\":', 'INRA6'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', '-'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.0104'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Manchega sheep.'], ['Breeds associated:\\n  \\nManchega'], ['Design:', 'A daughter design was used.  Animals were genotyped for polymorphisms in ACP6, CGN, ANXA9, and SLC27A3 and analyzed for milk traits.'], ['Analysis:', 'The general linear model procedure was used.'], ['Software:', 'CRIMAP, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Calvo JH, Mart�nez-Royo A, Beattie AE, Dodds KG, Marcos-Carcavilla A, Serrano M'], ['Affiliation:', 'Unidad de Tecnologia en Produccion Animal, CITA-Gobierno de Aragon, Zaragoza, Spain'], ['Title:', 'Fine mapping of genes on sheep chromosome 1 and their association with milk traits'], ['Journal:', 'Animal Genetics, 2006, 37(3): 205-10'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '1'], ['QTL Peak Location:', '124.3 (cM)'], ['QTL Span:', '113.8-124.5 (cM)93.1-97.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'AE57'], ['Peak:', ''], ['Lower, \"Significant\":', 'BMS482'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', '-'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.0275'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Manchega sheep.'], ['Breeds associated:\\n  \\nManchega'], ['Design:', 'A daughter design was used.  Animals were genotyped for polymorphisms in ACP6, CGN, ANXA9, and SLC27A3 and analyzed for milk traits.'], ['Analysis:', 'The general linear model procedure was used.'], ['Software:', 'CRIMAP, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Calvo JH, Mart�nez-Royo A, Beattie AE, Dodds KG, Marcos-Carcavilla A, Serrano M'], ['Affiliation:', 'Unidad de Tecnologia en Produccion Animal, CITA-Gobierno de Aragon, Zaragoza, Spain'], ['Title:', 'Fine mapping of genes on sheep chromosome 1 and their association with milk traits'], ['Journal:', 'Animal Genetics, 2006, 37(3): 205-10'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '1'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '255.51-255.51 (cM)216.4-216.4 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs413812442'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.030'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra dairy sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'CRIMAP, R software, QTLmap, GridQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-G�mez E, Guti�rrez-Gil B, Sahana G, S�nchez JP, Bay�n Y, Arranz JJ'], ['Affiliation:', 'Departmento Producci�n Animal, Universidad de Le�n, Le�n, Spain'], ['Title:', 'GWA analysis for milk production traits in dairy sheep and genetic support for a QTN influencing milk protein percentage in the LALBA gene'], ['Journal:', 'PloS one, 2012, 7(10): e47782'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '2'], ['QTL Peak Location:', '206 (cM)'], ['QTL Span:', '171.7-247.7 (cM)143.9-204.4 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'CSRD54'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Suggestive'], ['', 'LOD_score', '1.8'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Awassi x Merino sheep.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 200 microsatellite markers and analyzed for milk production traits.'], ['Analysis:', 'A maximum likelihood procedure (QTL-MLE) and a regression analysis for half-sib design were used.'], ['Software:', 'R software, QTL Express'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Raadsma HW, Jonas E, McGill D, Hobbs M, Lam MK, Thomson PC'], ['Affiliation:', 'ReproGen - Animal Bioscience Group, Faculty of Veterinary Science, University of Sydney, Camden NSW 2570, Australia'], ['Title:', 'Mapping Quantitative Trait Loci (QTL) in Sheep. II. Meta-assembly and identification of novel QTL for milk production traits in sheep.'], ['Journal:', 'Genetics, selection, evolution : GSE, 2009, 41: 45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '2'], ['QTL Peak Location:', '238.81 (cM)'], ['QTL Span:', '221.5-236.4 (cM)178.8-198.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BMS1126'], ['Peak:', ''], ['Lower, \"Significant\":', 'BMS2626'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '1.61'], ['F_Stat', '7.78'], ['P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', '26.98'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '2'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '69.74-69.74 (cM)55.2-55.2 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs405020493'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.013'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra dairy sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'CRIMAP, R software, QTLmap, GridQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-G�mez E, Guti�rrez-Gil B, Sahana G, S�nchez JP, Bay�n Y, Arranz JJ'], ['Affiliation:', 'Departmento Producci�n Animal, Universidad de Le�n, Le�n, Spain'], ['Title:', 'GWA analysis for milk production traits in dairy sheep and genetic support for a QTN influencing milk protein percentage in the LALBA gene'], ['Journal:', 'PloS one, 2012, 7(10): e47782'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '2'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '218.39-218.39 (cM)172.5-172.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs407624446'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.017'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra dairy sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'CRIMAP, R software, QTLmap, GridQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-G�mez E, Guti�rrez-Gil B, Sahana G, S�nchez JP, Bay�n Y, Arranz JJ'], ['Affiliation:', 'Departmento Producci�n Animal, Universidad de Le�n, Le�n, Spain'], ['Title:', 'GWA analysis for milk production traits in dairy sheep and genetic support for a QTN influencing milk protein percentage in the LALBA gene'], ['Journal:', 'PloS one, 2012, 7(10): e47782'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '2'], ['QTL Peak Location:', '68.17 (cM)'], ['QTL Span:', '65.81-70.52 (cM)52.0-55.8 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs419504162'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs408563812'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.001'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '3'], ['QTL Peak Location:', '88.2 (cM)'], ['QTL Span:', '73.5-113.0 (cM)41.4-76.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'DIK4796'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '2.3'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Awassi x Merino sheep.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 200 microsatellite markers and analyzed for milk production traits.'], ['Analysis:', 'A maximum likelihood procedure (QTL-MLE) and a regression analysis for half-sib design were used.'], ['Software:', 'R software, QTL Express'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Raadsma HW, Jonas E, McGill D, Hobbs M, Lam MK, Thomson PC'], ['Affiliation:', 'ReproGen - Animal Bioscience Group, Faculty of Veterinary Science, University of Sydney, Camden NSW 2570, Australia'], ['Title:', 'Mapping Quantitative Trait Loci (QTL) in Sheep. II. Meta-assembly and identification of novel QTL for milk production traits in sheep.'], ['Journal:', 'Genetics, selection, evolution : GSE, 2009, 41: 45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '3'], ['QTL Peak Location:', '130.96 (cM)'], ['QTL Span:', '130.93-130.99 (cM)97.0-97.0 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs162300018'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs409508507'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '4'], ['QTL Peak Location:', '114.67 (cM)'], ['QTL Span:', '114.67-114.67 (cM)92.5-92.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs420693815'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', '-'], ['Test base:', '-'], ['Threshold significance level:', 'Suggestive'], ['', 'P_values', '0.053'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'LEP (leptin)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Egyptian Barki sheep.'], ['Breeds associated:\\n  \\nBarki'], ['Design:', 'Animals were genotyped for SNPs in eight candidate genes and analyzed for growth and milk production traits.'], ['Analysis:', 'The general linear model of ANOVA was used for association analysis.'], ['Software:', 'SPSS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Abousoliman I, Reyer H, Oster M, Muráni E, Mourad M, Rashed MA, Mohamed I, Wimmers K'], ['Affiliation:', 'Faculty of Agricultural and Environmental Sciences, University of Rostock, Justus-von-Liebig-Weg 7, 18059 Rostock, Germany'], ['Title:', 'Analysis of Candidate Genes for Growth and Milk Performance Traits in the Egyptian Barki Sheep'], ['Journal:', 'Animals : an open access journal from MDPI, 2020, 10(2):E197'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '6'], ['QTL Peak Location:', '119.8 (cM)'], ['QTL Span:', '129.96-130.16 (cM)100.4-100.6 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.035'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'A daughter design was used.  Animals were genotyped for 11 microsatellite markers on chromosome 6 and analyzed for milk production traits.'], ['Analysis:', 'Multiple-marker regression analysis was used.'], ['Software:', 'PEST, CRIMAP'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Diez-Tasc�n C, Bay�n Y, Arranz JJ, De La Fuente F, San Primitivo F'], ['Affiliation:', 'Departamento de Producci�n Animal, Universidad de Le�n, Spain'], ['Title:', 'Mapping quantitative trait loci for milk production traits on ovine chromosome 6'], ['Journal:', 'J Dairy Res, 2001, 68(3): 389-97'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '6'], ['QTL Peak Location:', '59 (cM)'], ['QTL Span:', '57.41-57.61 (cM)42.0-42.2 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'BM143'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were from an Awassi x Hungarian Merino cross.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 13 microsatellite markers on chromosome 6 and analyzed for milk traits.'], ['Analysis:', 'The general linear model procedure was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Arnyasi M, Koml�si I, Lien S, Czegl�di L, Nagy S, J�vor A'], ['Affiliation:', 'Institute of Animal Sciences, Centre of Agricultural Sciences and Engineering, University of Debrecen, Debrecen, Hungary'], ['Title:', 'Searching for DNA markers for milk production and composition on chromosome 6 in sheep.'], ['Journal:', 'Journal of animal breeding and genetics, 2009, 126(2): 142-7'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '6'], ['QTL Peak Location:', '59 (cM)'], ['QTL Span:', '57.41-57.61 (cM)42.0-42.2 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'BM143'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were from an Awassi x Hungarian Merino cross.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 13 microsatellite markers on chromosome 6 and analyzed for milk traits.'], ['Analysis:', 'The general linear model procedure was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Arnyasi M, Koml�si I, Lien S, Czegl�di L, Nagy S, J�vor A'], ['Affiliation:', 'Institute of Animal Sciences, Centre of Agricultural Sciences and Engineering, University of Debrecen, Debrecen, Hungary'], ['Title:', 'Searching for DNA markers for milk production and composition on chromosome 6 in sheep.'], ['Journal:', 'Journal of animal breeding and genetics, 2009, 126(2): 142-7'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '6'], ['QTL Peak Location:', '49.9 (cM)'], ['QTL Span:', '48.33-48.53 (cM)68.7-68.8 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Comparison-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.024'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'ABCG2 (ATP-binding cassette, sub-family G (WHITE), member 2)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped for SNPs in DGAT1, GHR, ABCG2, and SPP1 and analyzed for milk production traits.'], ['Analysis:', 'A mixed model was used.'], ['Software:', 'R software'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-Fern�ndez M, Guti�rrez-Gil B, S�nchez JP, Mor�n JA, Garc�a-G�mez E, Alvarez L, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, Spain'], ['Title:', 'The role of bovine causal genes underlying dairy traits in Spanish Churra sheep'], ['Journal:', 'Animal Genetics, 2011, 42(4):415-20'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '6'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)98.9-99.1 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs412495304'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs425692142'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '7'], ['QTL Peak Location:', '60.46 (cM)'], ['QTL Span:', '60.41-60.50 (cM)40.7-40.8 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs410734119'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs409755708'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '9'], ['QTL Peak Location:', '16.6 (cM)'], ['QTL Span:', '16.6-16.6 (cM)13.4-13.4 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BM757'], ['Peak:', ''], ['Lower, \"Significant\":', 'ILST39A'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', '-'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.014'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'DGAT1 (diacylglycerol O-acyltransferase 1)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Altamurana, Gentile di Puglia, and Sarda sheep.  To confirm association, a resource population of Sarda x Lacaune backcross ewes with Sarda rams was used.'], ['Breeds associated:\\n  \\nSardianian'], ['Design:', 'Animals were genotyped for five SNPs in DGAT1 and analyzed for milk traits.'], ['Analysis:', 'The mixed procedure and ANOVA were used.'], ['Software:', 'SAS, MTDFREML'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Scat� MC, Napolitano F, Casu S, Carta A, De Matteis G, Signorelli F, Annicchiarico G, Catillo G, Moioli B'], ['Affiliation:', 'CRA-PCM, via Salaria 31, 00016 Monterotondo, Italy'], ['Title:', 'Ovine acyl CoA:diacylglycerol acyltransferase 1- molecular characterization, polymorphisms and association with milk traits'], ['Journal:', 'Animal Genetics, 2009, 40(5): 737-42'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '11'], ['QTL Peak Location:', '33 (cM)'], ['QTL Span:', '23.6-66.6 (cM)13.6-38.1 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'HEL10'], ['Peak:', ''], ['Lower, \"Significant\":', 'BM17132'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '2.6'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were from crosses between Awassi and Merino sheep.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 189 microsatellite markers and analyzed for lactation persistency and extended lactation traits.'], ['Analysis:', 'Two methods were used for QTL detection: a maximum likelihood procedure and regression analysis for a half-sib design.'], ['Software:', 'R software, QTL Express'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Jonas E, Thomson PC, Hall EJ, McGill D, Lam MK, Raadsma HW.'], ['Affiliation:', 'ReproGen-Animal Bioscience Group, Faculty of Veterinary Science, University of Sydney, 425 Werombi Road, Camden, NSW 2570, Australia'], ['Title:', 'Mapping quantitative trait loci (QTL) in sheep. IV. Analysis of lactation persistency and extended lactation traits in sheep'], ['Journal:', 'Genet Sel Evol., 2011, 43:22'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '11'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '83.71-83.71 (cM)47.5-47.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs397514093'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'GH (growth hormone)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian'], ['Design:', 'Animals were genotyped for SNPs in the GH gene and analyzed for milk traits.'], ['Analysis:', 'A mixed model was used.'], ['Software:', 'SAS'], ['Notes:', 'One of the SNPs studied (ss748770547) is on chromosome 1, while GH is on chromosome 11. Associations for this SNP were not entered.'], ['Links:', 'Edit']], [[], ['Authors:', 'Dettori ML, Pazzola M, Pira E, Paschino P, Vacca GM'], ['Affiliation:', 'Dipartimento di Medicina Veterinaria,Universit� degli Studi di Sassari,via Vienna,2 07100 Sassari,Italy'], ['Title:', 'The sheep growth hormone gene polymorphism and its effects on milk traits'], ['Journal:', 'The Journal of dairy research, 2015, 82(2): 169-76'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '11'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '83.70-83.70 (cM)47.5-47.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs397514076'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'GH (growth hormone)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian'], ['Design:', 'Animals were genotyped for SNPs in the GH gene and analyzed for milk traits.'], ['Analysis:', 'A mixed model was used.'], ['Software:', 'SAS'], ['Notes:', 'One of the SNPs studied (ss748770547) is on chromosome 1, while GH is on chromosome 11. Associations for this SNP were not entered.'], ['Links:', 'Edit']], [[], ['Authors:', 'Dettori ML, Pazzola M, Pira E, Paschino P, Vacca GM'], ['Affiliation:', 'Dipartimento di Medicina Veterinaria,Universit� degli Studi di Sassari,via Vienna,2 07100 Sassari,Italy'], ['Title:', 'The sheep growth hormone gene polymorphism and its effects on milk traits'], ['Journal:', 'The Journal of dairy research, 2015, 82(2): 169-76'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '12'], ['QTL Peak Location:', '37.73 (cM)'], ['QTL Span:', '23.7-54.2 (cM)18.1-37.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BM4025'], ['Peak:', ''], ['Lower, \"Significant\":', 'CSSM03'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '1.11'], ['F_Stat', '5.12'], ['P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', '-172.87'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '12'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '91.57-91.57 (cM)68.0-68.0 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs423175848'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.009'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra dairy sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'CRIMAP, R software, QTLmap, GridQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-G�mez E, Guti�rrez-Gil B, Sahana G, S�nchez JP, Bay�n Y, Arranz JJ'], ['Affiliation:', 'Departmento Producci�n Animal, Universidad de Le�n, Le�n, Spain'], ['Title:', 'GWA analysis for milk production traits in dairy sheep and genetic support for a QTN influencing milk protein percentage in the LALBA gene'], ['Journal:', 'PloS one, 2012, 7(10): e47782'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '13'], ['QTL Peak Location:', '20.25 (cM)'], ['QTL Span:', '20.23-20.28 (cM)13.0-13.1 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs404097073'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs418419846'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '13'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)19.4-20.1 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs401187348'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs406859166'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '13'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)20.4-20.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs418387788'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs408778222'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '13'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)48.8-49.7 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs417470080'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs412097174'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '13'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)49.0-49.7 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs422598859'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs412097174'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '14'], ['QTL Peak Location:', '57.9 (cM)'], ['QTL Span:', '64.09-64.29 (cM)39.9-40.0 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.02'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'LCAT (lecithin-cholesterol acyltransferase)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Altamurana, Gentile di Puglia, and Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian, Gentile di Puglia, Altamurana'], ['Design:', 'Animals were genotyped for 41 SNP in 16 genes and analyzed for milk traits.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Cris� A, Marchitelli C, Pariset L, Contarini G, Signorelli F, Napolitano F, Catillo G, Valentini A, Moioli B'], ['Affiliation:', 'Consiglio per la Ricerca e la Sperimentazione in Agricoltura, 00015 Monterotondo, Italy'], ['Title:', 'Exploring polymorphisms and effects of candidate genes on milk fat quality in dairy sheep'], ['Journal:', 'J Dairy Sci, 2010, 93(8):3834-45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '14'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)15.0-15.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs418617556'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs406088936'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '14'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)15.0-15.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs418617556'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs423521812'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '14'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)15.0-15.7 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs401264364'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs426280088'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '14'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)18.0-18.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs413432980'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs404302625'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '15'], ['QTL Peak Location:', '73.5 (cM)'], ['QTL Span:', '80.34-82.05 (cM)57.0-57.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.03'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Altamurana, Gentile di Puglia, and Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian, Gentile di Puglia, Altamurana'], ['Design:', 'Animals were genotyped for 41 SNP in 16 genes and analyzed for milk traits.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Cris� A, Marchitelli C, Pariset L, Contarini G, Signorelli F, Napolitano F, Catillo G, Valentini A, Moioli B'], ['Affiliation:', 'Consiglio per la Ricerca e la Sperimentazione in Agricoltura, 00015 Monterotondo, Italy'], ['Title:', 'Exploring polymorphisms and effects of candidate genes on milk fat quality in dairy sheep'], ['Journal:', 'J Dairy Sci, 2010, 93(8):3834-45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '16'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '38.08-38.08 (cM)32.2-32.2 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs404584067'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.007'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra dairy sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'CRIMAP, R software, QTLmap, GridQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garc�a-G�mez E, Guti�rrez-Gil B, Sahana G, S�nchez JP, Bay�n Y, Arranz JJ'], ['Affiliation:', 'Departmento Producci�n Animal, Universidad de Le�n, Le�n, Spain'], ['Title:', 'GWA analysis for milk production traits in dairy sheep and genetic support for a QTN influencing milk protein percentage in the LALBA gene'], ['Journal:', 'PloS one, 2012, 7(10): e47782'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '16'], ['QTL Peak Location:', '13.53 (cM)'], ['QTL Span:', '11.86-15.19 (cM)10.1-12.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs409681162'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs414277369'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.003'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '16'], ['QTL Peak Location:', '14.13 (cM)'], ['QTL Span:', '11.86-16.40 (cM)10.1-13.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs409681162'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs412655645'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '16'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)41.5-41.6 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs412107424'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs161154668'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '17'], ['QTL Peak Location:', '52.31 (cM)'], ['QTL Span:', '28.39-76.23 (cM)15.8-42.4 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs408339501'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs402945893'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '18'], ['QTL Peak Location:', '40.88 (cM)'], ['QTL Span:', '22.3-56 (cM)10.6-33.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BM3413'], ['Peak:', ''], ['Lower, \"Significant\":', 'UWCA4'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '1.07'], ['F_Stat', '5.02'], ['P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', '-33.83'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '18'], ['QTL Peak Location:', '40.88 (cM)'], ['QTL Span:', '22.3-56 (cM)10.6-33.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BM3413'], ['Peak:', ''], ['Lower, \"Significant\":', 'UWCA4'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '1.44'], ['F_Stat', '6.95'], ['P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', '-0.46'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '18'], ['QTL Peak Location:', '91.7 (cM)'], ['QTL Span:', '107.59-107.79 (cM)63.4-63.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.006'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Altamurana, Gentile di Puglia, and Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian, Gentile di Puglia, Altamurana'], ['Design:', 'Animals were genotyped for 41 SNP in 16 genes and analyzed for milk traits.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Cris� A, Marchitelli C, Pariset L, Contarini G, Signorelli F, Napolitano F, Catillo G, Valentini A, Moioli B'], ['Affiliation:', 'Consiglio per la Ricerca e la Sperimentazione in Agricoltura, 00015 Monterotondo, Italy'], ['Title:', 'Exploring polymorphisms and effects of candidate genes on milk fat quality in dairy sheep'], ['Journal:', 'J Dairy Sci, 2010, 93(8):3834-45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '18'], ['QTL Peak Location:', '91.7 (cM)'], ['QTL Span:', '107.59-107.79 (cM)63.4-63.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', ''], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'n/a'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.005'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Altamurana, Gentile di Puglia, and Sarda sheep.'], ['Breeds associated:\\n  \\nSardianian, Gentile di Puglia, Altamurana'], ['Design:', 'Animals were genotyped for 41 SNP in 16 genes and analyzed for milk traits.'], ['Analysis:', 'A linear mixed model was used.'], ['Software:', 'SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Cris� A, Marchitelli C, Pariset L, Contarini G, Signorelli F, Napolitano F, Catillo G, Valentini A, Moioli B'], ['Affiliation:', 'Consiglio per la Ricerca e la Sperimentazione in Agricoltura, 00015 Monterotondo, Italy'], ['Title:', 'Exploring polymorphisms and effects of candidate genes on milk fat quality in dairy sheep'], ['Journal:', 'J Dairy Sci, 2010, 93(8):3834-45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '19'], ['QTL Peak Location:', '57.44 (cM)'], ['QTL Span:', '56.39-58.50 (cM)47.5-49.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs413929589'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs411539281'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '0.031'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '19'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)40.2-41.1 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs421440887'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs410814232'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '19'], ['QTL Peak Location:', '0.00 (cM)'], ['QTL Span:', '0.00-0.00 (cM)46.5-46.5 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs410379548'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs428546783'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'CNV'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Experiment-wise'], ['Threshold significance level:', 'Significant'], ['', 'FDR', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip v2. A total of 52,413 SNPs were used for CNV detection and GWAS for milk production traits.'], ['Analysis:', 'A correlation/trend test model was used.'], ['Software:', 'SVS 8.7.0, SAS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Di Gerlando R, Sutera AM, Mastrangelo S, Tolone M, Portolano B, Sottile G, Bagnato A, Strillacci MG, Sardina MT'], ['Affiliation:', 'Università degli Studi di Palermo, Dipartimento di Scienze Agrarie, Alimentari e Forestali, Italy'], ['Title:', 'Genome-wide association study between CNVs and milk production traits in Valle del Belice sheep'], ['Journal:', 'PloS one, 2019, 14(4): e0215204'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '20'], ['QTL Peak Location:', '36.8 (cM)'], ['QTL Span:', '21.2-53.9 (cM)12.3-23.7 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'DYA'], ['Peak:', ''], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '2.0'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Awassi x Merino sheep.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 200 microsatellite markers and analyzed for milk production traits.'], ['Analysis:', 'A maximum likelihood procedure (QTL-MLE) and a regression analysis for half-sib design were used.'], ['Software:', 'R software, QTL Express'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Raadsma HW, Jonas E, McGill D, Hobbs M, Lam MK, Thomson PC'], ['Affiliation:', 'ReproGen - Animal Bioscience Group, Faculty of Veterinary Science, University of Sydney, Camden NSW 2570, Australia'], ['Title:', 'Mapping Quantitative Trait Loci (QTL) in Sheep. II. Meta-assembly and identification of novel QTL for milk production traits in sheep.'], ['Journal:', 'Genetics, selection, evolution : GSE, 2009, 41: 45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '20'], ['QTL Peak Location:', '39.82 (cM)'], ['QTL Span:', '0-69.5 (cM)2.4-38.4 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', ''], ['Lower, \"Significant\":', 'BM1818'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Chromosome-wise'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '1.14'], ['F_Stat', '5.56'], ['P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', '266.97'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '20'], ['QTL Peak Location:', '66.88 (cM)'], ['QTL Span:', '66.88-66.88 (cM)34.3-34.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs422713690'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'Association'], ['Model tested:', '-'], ['Test base:', '-'], ['Threshold significance level:', 'Suggestive'], ['', 'P_values', '0.052'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'PRL (prolactin)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Egyptian Barki sheep.'], ['Breeds associated:\\n  \\nBarki'], ['Design:', 'Animals were genotyped for SNPs in eight candidate genes and analyzed for growth and milk production traits.'], ['Analysis:', 'The general linear model of ANOVA was used for association analysis.'], ['Software:', 'SPSS'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Abousoliman I, Reyer H, Oster M, Muráni E, Mourad M, Rashed MA, Mohamed I, Wimmers K'], ['Affiliation:', 'Faculty of Agricultural and Environmental Sciences, University of Rostock, Justus-von-Liebig-Weg 7, 18059 Rostock, Germany'], ['Title:', 'Analysis of Candidate Genes for Growth and Milk Performance Traits in the Egyptian Barki Sheep'], ['Journal:', 'Animals : an open access journal from MDPI, 2020, 10(2):E197'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '21'], ['QTL Peak Location:', '43.59 (cM)'], ['QTL Span:', '43.59-43.59 (cM)28.9-28.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs398340969'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Polymorphic markers'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Comparison-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '1.73E-5'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'DCPS (decapping enzyme, scavenger)'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 37,228 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was employed for GWAS using repeated measures.'], ['Software:', 'GenABEL, RepeatABEL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Sutera A M, Riggio V, Mastrangelo S, Di Gerlando R, Sardina M T, Pong-Wong R, Tolone M, Portolano B'], ['Affiliation:', 'Dipartimento Scienze Agrarie, Alimentari e Forestali, University of Palermo, 90128, Palermo, Italy'], ['Title:', 'Genome-wide association studies for milk production traits in Valle del Belice sheep using repeated measures'], ['Journal:', 'Animal genetics, 2019, 50(3):311-314'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '22'], ['QTL Peak Location:', '10.68 (cM)'], ['QTL Span:', '10.64-10.72 (cM)6.5-6.6 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'rs402450066'], ['Peak:', ''], ['Lower, \"Significant\":', 'rs417784043'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'SNP'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '<0.05'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Spanish Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 43,784 SNPs were used for analysis.'], ['Analysis:', 'Linkage analysis and combined linkage disequilibrium linkage analysis were performed.'], ['Software:', 'QTLMap'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Garcia-G�mez E, Guti�rrez-Gil B, Suarez-Vega A, de la Fuente LF, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Le�n, Spain'], ['Title:', 'Identification of quantitative trait loci underlying milk traits in Spanish dairy sheep using linkage plus combined linkage disequilibrium and linkage analysis approaches'], ['Journal:', 'Journal of dairy science, 2013, 96(9): 6059-69'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '23'], ['QTL Peak Location:', '68.97 (cM)'], ['QTL Span:', '52.8-83.9 (cM)45.5-56.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'MCM136'], ['Peak:', ''], ['Lower, \"Significant\":', 'URB031'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Suggestive'], ['', 'P_values', '0.0670'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nNCBI GeneDB\\xa0\\xa0|\\xa0\\xa0Edit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Churra sheep.'], ['Breeds associated:\\n  \\nSpanish Churra'], ['Design:', 'A daughter design was used.  Animals were genotyped for 181 microsatellite markers and 1 SNP and analyzed for milk production traits.'], ['Analysis:', 'A multi-marker regression method was used.'], ['Software:', 'CRIMAP, HSQM'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Guti�rrez-Gil B, El-Zarei MF, Alvarez L, Bay�n Y, de la Fuente LF, San Primitivo F, Arranz JJ'], ['Affiliation:', 'Departamento de Producci�n Animal, Facultad de Veterinaria, Universidad de Le�n, 24071 Leon, Spain'], ['Title:', 'Quantitative trait loci underlying milk production traits in sheep.'], ['Journal:', 'Animal genetics, 2009, 40(4): 423-34'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '24'], ['QTL Peak Location:', '73 (cM)'], ['QTL Span:', '58.5-73 (cM)32.0-42.0 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'DIK5147'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', 'Genome-wise'], ['Threshold significance level:', 'Suggestive'], ['', 'LOD_score', '1.9'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Awassi x Merino sheep.'], ['Breeds associated:\\n  \\nAwassi, Merino'], ['Design:', 'Animals were genotyped for 200 microsatellite markers and analyzed for milk production traits.'], ['Analysis:', 'A maximum likelihood procedure (QTL-MLE) and a regression analysis for half-sib design were used.'], ['Software:', 'R software, QTL Express'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Raadsma HW, Jonas E, McGill D, Hobbs M, Lam MK, Thomson PC'], ['Affiliation:', 'ReproGen - Animal Bioscience Group, Faculty of Veterinary Science, University of Sydney, Camden NSW 2570, Australia'], ['Title:', 'Mapping Quantitative Trait Loci (QTL) in Sheep. II. Meta-assembly and identification of novel QTL for milk production traits in sheep.'], ['Journal:', 'Genetics, selection, evolution : GSE, 2009, 41: 45'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '24'], ['QTL Peak Location:', '14.46 (cM)'], ['QTL Span:', '0-23 (cM)2.6-12.3 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'BMS514'], ['Peak:', ''], ['Lower, \"Significant\":', 'BM4005'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Microsatellite'], ['Analysis type:', 'QTL'], ['Model tested:', 'Mendelian'], ['Test base:', '-'], ['Threshold significance level:', 'Significant'], ['', 'LOD_score', '2.37'], ['F_Stat', '11.73'], ['P_values', '<0.01'], ['Dominance effect:', 'n/a'], ['Additive effect:', '223.94'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were East Friesian x Dorset sheep.'], ['Breeds associated:\\n  \\nDorset, Friesian Milk'], ['Design:', 'Animals were genotyped for 120 microsatellite markers and analyzed for milk traits.'], ['Analysis:', 'A least-squares method was used.'], ['Software:', 'GRIDQTL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Mateescu R G, Thonney M L'], ['Affiliation:', 'Department of Animal Science, Oklahoma State University, Stillwater, OK 74078, USA'], ['Title:', 'Genetic mapping of quantitative trait loci for milk production in sheep.'], ['Journal:', 'Animal genetics, 2010, 41(5):460-6'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']], [[], ['Chromosome:', '26'], ['QTL Peak Location:', '16.02 (cM)'], ['QTL Span:', '16.02-16.02 (cM)9.9-9.9 (Mbp)'], ['', 'Upper, \"Suggestive\":', 'n/a'], ['Upper, \"Significant\":', 'n/a'], ['Peak:', 'rs417920940'], ['Lower, \"Significant\":', 'n/a'], ['Lower, \"Suggestive\":', 'n/a'], ['Marker type:', 'Polymorphic markers'], ['Analysis type:', 'Association'], ['Model tested:', 'Mendelian'], ['Test base:', 'Comparison-wise'], ['Threshold significance level:', 'Significant'], ['', 'P_values', '1.47E-5'], ['Dominance effect:', 'n/a'], ['Additive effect:', 'n/a'], ['Associated Gene:', 'n/a'], ['Links:\\xa0\\xa0\\nEdit\\xa0\\xa0|\\xa0\\xa0\\nMap view']], [[], ['Animals:', 'Animals were Valle del Belice sheep.'], ['Breeds associated:\\n  \\nValle del Belice'], ['Design:', 'Animals were genotyped using the Illumina OvineSNP50 BeadChip and analyzed for milk production traits. A total of 37,228 SNPs were used for analysis.'], ['Analysis:', 'A linear mixed model was employed for GWAS using repeated measures.'], ['Software:', 'GenABEL, RepeatABEL'], ['Notes:', ''], ['Links:', 'Edit']], [[], ['Authors:', 'Sutera A M, Riggio V, Mastrangelo S, Di Gerlando R, Sardina M T, Pong-Wong R, Tolone M, Portolano B'], ['Affiliation:', 'Dipartimento Scienze Agrarie, Alimentari e Forestali, University of Palermo, 90128, Palermo, Italy'], ['Title:', 'Genome-wide association studies for milk production traits in Valle del Belice sheep using repeated measures'], ['Journal:', 'Animal genetics, 2019, 50(3):311-314'], ['Links:', 'PubMed\\xa0\\xa0|\\xa0\\xa0Abstract\\xa0\\xa0   |\\xa0\\xa0List all QTL\\xa0\\xa0\\n   |\\xa0\\xa0Edit']]]\n",
      "168\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "datc=[]\n",
    "for j in range(0,len(LINK)):\n",
    "\n",
    "  #i)\n",
    "  r = requests.get(LINK[j])\n",
    "  x = BeautifulSoup(r.text)\n",
    "  #print(bs)\n",
    "  x=str(x)\n",
    "\n",
    "  #primera tabla:\n",
    "  inicio=\"\"\"<table border=\"2\" bordercolor=\"#F5F5cC\" cellpadding=\"3\" cellspacing=\"0\" width=\"100%\">\"\"\"\n",
    "  rini=len(inicio)\n",
    "  termino=\"\"\"</table>\"\"\"\n",
    "  rter=len(termino)\n",
    "  c=\"\"\n",
    "  c2=\"\"\n",
    "\n",
    "  for i in range(0,len(x)):\n",
    "    if x[i:i+rini]==inicio:\n",
    "      c=c+x[i:len(x)]\n",
    "      #print(c)\n",
    "      break\n",
    "      \n",
    "  for i in range(0,len(c)):\n",
    "    if c[i:i+rter]==termino:\n",
    "      c2=c2+c[0:i+8]\n",
    "      #print(c2)\n",
    "      break\n",
    "\n",
    "  #segunda tabla:\n",
    "  inicio2=\"\"\"<table border=\"2\" bordercolor=\"#feddfe\" cellpadding=\"2\" cellspacing=\"0\" width=\"380\">\"\"\"\n",
    "  rini2=len(inicio2)\n",
    "  termino2=\"\"\"</table>\"\"\"\n",
    "  rter2=len(termino2)\n",
    "  c3=\"\"\n",
    "  c4=\"\"\n",
    "  \n",
    "  for i in range(0,len(x)):\n",
    "    if x[i:i+rini2]==inicio2:\n",
    "      c3=c3+x[i:len(x)]\n",
    "      #print(c3)\n",
    "      break\n",
    "      \n",
    "  for i in range(0,len(c3)):\n",
    "    if c3[i:i+rter2]==termino2:\n",
    "      c4=c4+c3[0:i+8]\n",
    "      #print(c4)\n",
    "      break\n",
    "      \n",
    "  #tercera tabla:\n",
    "  inicio3=\"\"\"<table border=\"2\" bordercolor=\"#E8e8Ff\" cellpadding=\"2\" cellspacing=\"0\" width=\"380\">\"\"\"\n",
    "  rini3=len(inicio3)\n",
    "  termino3=\"\"\"</table>\"\"\"\n",
    "  rter3=len(termino3)\n",
    "  c5=\"\"\n",
    "  c6=\"\"\n",
    "  \n",
    "  for i in range(0,len(x)):\n",
    "    if x[i:i+rini3]==inicio3:\n",
    "      c5=c5+x[i:len(x)]\n",
    "      #print(c5)\n",
    "      break\n",
    "  \n",
    "  for i in range(0,len(c5)):\n",
    "    if c5[i:i+rter3]==termino3:\n",
    "      c6=c6+c5[0:i+8]\n",
    "      #print(c6)\n",
    "      break\n",
    "\n",
    "  trip=[]\n",
    "  trip.append(c2)\n",
    "  trip.append(c4)\n",
    "  trip.append(c6)\n",
    "  \n",
    "  #print(trip)\n",
    "\n",
    "  #datc=[]\n",
    "  for i in trip:\n",
    "    pri=[]\n",
    "    bs=BeautifulSoup(i, \"lxml\")\n",
    "    table_body=bs.find('table')\n",
    "    rows = table_body.find_all('tr')\n",
    "    for row in rows:\n",
    "      cols=row.find_all('td')\n",
    "      cols=[x.text.strip() for x in cols]\n",
    "      #print(cols)\n",
    "      pri.append(cols)\n",
    "      #print(pri)\n",
    "    datc.append(pri)\n",
    "\n",
    "print(datc)\n",
    "print(len(datc))\n",
    "\n",
    "datc[0][1][1]\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "colab": {
   "collapsed_sections": [],
   "name": "Untitled17.ipynb",
   "provenance": []
  },
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
